app/frontend/src/views/form/Manage.vue

```diff
@@ -117,16 +117,16 @@
               }"
             >
               <v-btn v-if="canCreateDesign" color="primary" text small>
-                <v-icon class="mr-1 ml-5">edit</v-icon>
+                <v-icon class="mr-1">edit</v-icon>
                 <span>Edit Current Design</span>
               </v-btn>
             </router-link>
           </p>
 
           <BaseInfoCard>
```

Comment:
this notice could only be shown if 'canCreateDesign'. maybe something i'll run by Mina.

Reply:
I'm doing a list of permission checks (and fixes if any required) in 1643, will note this in there